client/lib/two-step-authorization/index.js

```diff
@@ -65,6 +67,70 @@ TwoStepAuthorization.prototype.fetch = function( callback ) {
 	);
 };
 
+TwoStepAuthorization.prototype.postLoginRequest = function( endpoint, data ) {
+	if ( ! this.getTwoStepWebauthnNonce() ) {
+		return Promise.reject( 'Invalid nonce' );
+	}
+
+	const url = 'https://wordpress.com/wp-login.php?action=' + endpoint;
+	// eslint-disable-next-line no-undef
+	const formData = new FormData();
```

Comment:
this should probably be `window.FormData`, which would let us drop the eslint disable on the previous line.

Reply:
Addressed in 89714c680eec7ca23b470420219f92403d931bb6